GOVERNANCE.md

```diff
@@ -33,7 +33,7 @@ Contributors are eligible to vote on Technical Committee elections. Contributors
 
 Being contributor is a prerequisite for becoming a maintainer to the projects they are contributing to.
 In order for a contributor to become a maintainer, the individual must be nominated by one of the existing maintainers with a reference to the individual's history of contributions to the project.
-The nomination is done as a pull request to the *CODEOWNERS* file in corresponding project's git repository. The approval must be done on the pull request by project's maintainers.
+The nomination is discussed and voted on in the maintainer discussion channel. If approved, the nominated contributor must be added to the maintainer list.
```

Comment:
Should we say anything about what "approved" means in this context? Unanimous approval from existing maintainers? Or a majority? Or at least two approvals like for PRs?

Reply:
I've rephrased this text now